integration_tests/integration/incidentDetailsEdit.cy.ts

```diff
@@ -277,5 +279,81 @@ context('Incident details (edit) - statement incomplete', () => {
         expect(loc.pathname).to.eq(adjudicationUrls.detailsOfOffence.urls.start(34))
       })
     })
+    it('should check that DICOVERY button to Yes if DISCOVERY date same as INCIDENT date ', () => {
+      cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
+      const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="Yes"]').should('be.checked')
+    })
+    it('should fail to submit as Hours is cleared with appropriate message', () => {
+      cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
+      const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+      incidentDetailsPage.locationSelector().select('Workshop 2')
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="No"]').click()
+      incidentDetailsPage.timeInputHoursDiscovery().clear()
+      incidentDetailsPage.submitButton().click()
+      incidentDetailsPage
+        .errorSummary()
+        .find('li')
+        .then($errors => {
+          expect($errors.get(0).innerText).to.contain('Enter the time of the discovery')
+        })
+    })
+    it('should submit successfully if No DISCOVERY selected and time edited', () => {
+      cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
+      const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="No"]').click()
+      incidentDetailsPage.timeInputHoursDiscovery().clear()
+      incidentDetailsPage.timeInputHoursDiscovery().type('13')
+      incidentDetailsPage.timeInputMinutesDiscovery().clear()
+      incidentDetailsPage.timeInputMinutesDiscovery().type('00')
+      incidentDetailsPage.locationSelector().select('Workshop 2')
```

Comment:
Same as the other comment - I don't think you need to change the location here as it doesn't impact what you're testing